tests/versioning.go

```diff
@@ -3736,15 +3741,24 @@ func (s *VersioningIntegSuite) validateWorkflowBuildId(
 	newVersioning bool,
 	expectingStamp bool,
 	expectedInheritedBuildId string,
+	extraSearchAttrBuildIds []string,
 ) {
 	dw, err := s.sdkClient.DescribeWorkflowExecution(ctx, wfId, runId)
 	s.NoError(err)
+	var searchAttr []string
+	err = converter.GetDefaultDataConverter().FromPayload(dw.GetWorkflowExecutionInfo().GetSearchAttributes().GetIndexedFields()["BuildIds"], &searchAttr)
```

Comment:
use `searchattribute.DecodeValue(..., enumspb.INDEXED_VALUE_TYPE_KEYWORD_LIST, true)` like in MutableStateImpl.loadBuildIds?